webpack.config.babel.js

```diff
@@ -0,0 +1,46 @@
+import webpack from 'webpack';
+import path from 'path';
+
+const { NODE_ENV } = process.env;
+
+const plugins = [
+  new webpack.optimize.OccurenceOrderPlugin(),
+  new webpack.DefinePlugin({
+    'process.env.NODE_ENV': JSON.stringify(NODE_ENV),
+  }),
+];
+
+const filename = `redux-actions${NODE_ENV === 'production' ? '.min' : ''}.js`;
+
+NODE_ENV === 'production'  && plugins.push(
```

Comment:
`NODE_ENV === 'production'` is being used twice. Not a big deal, but what do you think about DRYing it up with a variable

```js
const isProduction = NODE_ENV === 'production'
```

Also, there's an extra space before `&& plugins.push(`.